src/Informer/CMakeLists.txt

```diff
@@ -3,20 +3,31 @@
 
 set(LIBRARY Informer)
 
-set(LIBRARY_SOURCES
+add_spectre_library(${LIBRARY})
+
+spectre_target_sources(
+  ${LIBRARY}
+  PRIVATE
   Informer.cpp
   Verbosity.cpp
   ${CMAKE_BINARY_DIR}/Informer/InfoAtCompile.cpp
```

Comment:
The correct way to handle this source file isn't obvious to me, and also for `InfoAtLink.cpp`. Do we need a way to associate uncompiled source files with the library? Do we need to move these to a different directory? @nilsdeppe I'd like to know your thoughts on this one.

Reply:
I think `InfoAtCompile.cpp` is handle correctly this way. Basically, we assume that `InfoAtCompile.cpp` will be generated before this is called. Maybe adding a comment that this needs to be true?

`InfoAtlink.cpp` is some very mystical think that is basically just used to inject some info late in the build process. I think we can safely ignore if for these purposes. It's unlikely to have incorrect includes and the way we track the dependency is through the header anyway. `InfoAtCompile.cpp` does need to be tracked correctly to be associated with this library.

Reply:
Thanks @nilsdeppe for your thoughts on this. I'll update the PR tomorrow to add some comments to acknowledge (and hopefully clarify) the special status of these two sources files.

Reply:
Okay, sounds good!